enterprise/backend/src/metabase_enterprise/serialization/v2/ingest.clj

```diff
@@ -61,12 +62,17 @@
 (defn- ingest-all [^File root-dir]
   ;; This returns a map {unlabeled-hierarchy [original-hierarchy File]}.
   (into {} (for [^File file (file-seq root-dir)
-                 :when      (and (.isFile file)
-                                 (str/ends-with? (.getName file) ".yaml")
-                                 (let [rel (.relativize (.toPath root-dir) (.toPath file))]
-                                   (-> rel (.subpath 0 1) (.toString) legal-top-level-paths)))
+                 :when (and (.isFile file)
+                            (not (str/starts-with? (.getName file) "."))
```

Comment:
I think this line doesn't have any test coverage by mistake. [L869](https://github.com/metabase/metabase/pull/41887/files?w=1#diff-4a66cb4cde8705cf92729e19d64fdac751f79b55d1870d54041e9a8dbe18cbb0R869) looks like it's meant to add coverage, but it's not actually testing that the hidden files are skipped. It just tests that there were no exceptions.

Reply:
done in https://github.com/metabase/metabase/pull/41887/commits/aa7e40f4acf1fdd1ac7d49fddd95857289b22e15